tests/Formatters/AbstractFormatterTests.cs

```diff
@@ -50,21 +49,25 @@ static AbstractFormatterTest()
                 LazyThreadSafetyMode.ExecutionAndPublication);
         }
 
-        protected virtual string DefaultFilePathPrefix { get; } = "Test";
+        protected virtual string DefaultFilePathPrefix => "Test";
 
-        protected virtual string DefaultTestProjectName { get; } = "TestProject";
+        protected virtual string DefaultTestProjectName => "TestProject";
 
-        protected virtual string DefaultTestProjectPath => "." + Path.DirectorySeparatorChar + DefaultTestProjectName + "." + DefaultFileExt + "proj";
+        protected virtual string DefaultFolderPath => Path.DirectorySeparatorChar + DefaultTestProjectName + Path.DirectorySeparatorChar;
```

Comment:
Can we `Path.Combine` for all of these cases? Why append the slashes manually?

Reply:
The beginning separator is important because I need the path to appear rooted. Using Path.Combine elsewhere is a thing i can do tho.